improver/cli/apply_emos_coefficients.py

```diff
@@ -133,11 +142,12 @@ def process(
     """
     import warnings
 
+    import iris
     import numpy as np
 
     from improver.calibration.ensemble_calibration import ApplyEMOS
 
-    if cube.name() in ["emos_coefficients", "land_binary_mask"]:
+    if not isinstance(cube, iris.cube.Cube) or cube.name() == "land_binary_mask":
```

Comment:
Will the `cube` type error not be picked up automatically by Clize?

Reply:
Yes, I removed the check on the type.